### DISCUSSION TITLE
[Portal engine] Data from block in localized fields are not shown in portal

### DISCUSSION THREAD

**Discussion by Maksonis** [pimcore/pimcore#13159] [Q&A / Support] [answered]
Hi,

when you add block in localized fields data is not shown in portal detail page.

Steps:

- add it like this:
![image](https://user-images.githubusercontent.com/17277953/190330993-5aed7acf-1a6e-47a2-9582-acc83d0d6e64.png)

- add it in Portal custom layout
![image](https://user-images.githubusercontent.com/17277953/190331133-9443a3db-554b-449d-b9dc-3abcdf2d4bcb.png)

- fill data in Car object
![image](https://user-images.githubusercontent.com/17277953/190331266-6f023539-13c7-4ce7-a3cc-162b17d33a75.png)

- always Empty data is shown:
![image](https://user-images.githubusercontent.com/17277953/190331354-4a6efbe4-f8cc-4c5a-a849-2a3add065695.png)

Is that a bug, or that currently not supported in portal ?

**Comment by Maksonis** [comment]
Any news here ?

**Reply by kingjia90** [reply]
I am sorry for that, it seems that this went overlooked because it is in the discussion area and i don't regularly check every thread.
It looks like related/follow-up of https://github.com/pimcore/pimcore/discussions/12978 so i would be glad to double-check this case as well.

I am going to move this discussion to an issue but before that, just wanted to make sure you have access to Portal Engine Repository, since i have noticed that you posted other portal engine related issues in core discussion area.
If not, please contact [enterprise-support@pimcore.com](mailto:enterprise-support@pimcore.com).

**Reply by Maksonis** [reply]
@kingjia90 thank you. I think it's opposite then https://github.com/pimcore/pimcore/discussions/12978 since there is localized fields inside block, and here is block inside localized field :)

Yeah, would be good since I don't have access. My company has. Since I work a lot of with the portal bundle and all enterprise bundles, maybe I can fix some small issues by myself.

**Comment by kingjia90** [comment, marked as answer]
Just wanted to update that the [bugfix](https://github.com/pimcore/portal-engine/pull/248) would be released today in the afternoon as version 2.4.2.
Thank you again for reporting!
